src/__tests__/utils/password.test.ts

```diff
@@ -0,0 +1,18 @@
+import { checkHash, hashFunction } from '../../utils/password';
+
+describe('testing the hashFunction', () => {
+  it('should compare the password hashed and without', () => {
```

Comment:
this is a nitpick, but hashing is technically not exclusive to passwords, so we should use `'secret'` instead of `'password'`

Reply:
Cool, I didn't know that.